.drone.star

```diff
@@ -0,0 +1,100 @@
+OC_CI_PHP = "owncloudci/php:%s"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+PLUGINS_GITHUB_RELEASE = "plugins/github-release"
+
+DEFAULT_PHP_VERSION = "8.1"
+
+dir = {
+    "base": "/var/www/owncloud",
+}
+
+config = {
+    "branches": [
+        "main",
+    ],
+    "codestyle": True,
+    "phpstan": True,
+    "phan": {
+        "multipleVersions": {
+            "phpVersions": [
+                DEFAULT_PHP_VERSION,
+                "8.2",
+            ],
+        },
+    },
+    "phpunit": True,
+}
+
+
+def main(ctx):
+    return tests(
+        ctx,
+        [
+            ["codestyle", "make test-php-style"],
+            ["phpstan", "make test-php-phpstan"],
+            ["phan", "make test-php-phan"],
+        ],
+    ) + phpunit(ctx, [8.1, 8.2])
+
+
+def tests(ctx, tests):
+    pipelines = []
+    for index, test in enumerate(tests):
+        if test[0] not in config:
+            return pipelines
```

Comment:
We don't want to stop the whole loop when we find the first thing that is not in the config.
Just that single pipeline needs to be skipped.